test/metabase/api/pulse_test.clj

```diff
@@ -963,90 +969,28 @@
                                                           :card_id      card-id
                                                           :target       [:variable [:template-tag "x"]]}]}]
       (mt/with-fake-inbox
-        (is (= {:ok true}
-               (mt/user-http-request :rasta :post 200 "pulse/test" {:name          "Daily Sad Toucans"
-                                                                    :dashboard_id  dashboard-id
-                                                                    :cards         [{:id                card-id
-                                                                                     :include_csv       false
-                                                                                     :include_xls       false
-                                                                                     :dashboard_card_id nil}]
-                                                                    :channels      [{:enabled       true
-                                                                                     :channel_type  "email"
-                                                                                     :schedule_type "daily"
-                                                                                     :schedule_hour 12
-                                                                                     :schedule_day  nil
-                                                                                     :recipients    [(mt/fetch-user :rasta)]}]
-                                                                    :skip_if_empty false})))
-        (is (= (mt/email-to :rasta {:subject "Daily Sad Toucans"
-                                    :body    {"Daily Sad Toucans" true}
-                                    :bcc?    true})
-               (mt/regex-email-bodies #"Daily Sad Toucans")))))))
-
-(deftest send-placeholder-card-test-pulse-test
```

Comment:
this is an old use case with pulse, it's no longer needed because this test endpoint only works on dashboard, and on dashboard all cards are saved.